packages/cli/commands/sandbox/create.js

```diff
@@ -118,18 +120,37 @@ exports.handler = async options => {
       name: sandboxName,
     })
   );
-
-  const result = await createSandbox(accountId, sandboxName).then(
-    ({ name, sandboxHubId }) => {
-      logger.success(
-        i18n(`${i18nKey}.success.create`, {
-          name,
-          sandboxHubId,
-        })
+  let result;
+  try {
+    result = await createSandbox(accountId, sandboxName).then(
+      ({ name, sandboxHubId }) => {
+        logger.success(
+          i18n(`${i18nKey}.success.create`, {
+            name,
+            sandboxHubId,
+          })
+        );
+        return { name, sandboxHubId };
+      }
+    );
+  } catch (err) {
+    if (
+      err.error &&
+      err.error.category &&
+      err.error.category === 'MISSING_SCOPES'
+    ) {
+      const websiteOrigin = getHubSpotWebsiteOrigin(env);
+      const url = `${websiteOrigin}/personal-access-key/${accountId}`;
+      logger.info(
+        `
+          Verify that the personal access key used has the Sandboxes scope by navigating to "${url}".
```

Comment:
is this the error message? I would say "The personal access key you provided doesn't give access to sandbox permissions. Deactivate the key and generate a new one with the correct permissions. After that, run `hs auth` again and enter the new key."

Reply:
Yep this is a permissions error message here - I'll get this change rolled in!

Reply:
After checking in with Erin, we adjusted to use `The personal access key you provided doesn't include sandbox permissions` in place of the "ERROR" line. Also adjusted `Visit` to `Go to` for the URL line.